src/index.js

```diff
@@ -1,5 +1,15 @@
-import './style.css';
+import LocalStorage from './modules/storage.js';
+import Data from './modules/data.js';
 
-import renderTasks from './modules/data.js';
+const storage = new LocalStorage('tasks');
 
-window.addEventListener('load', renderTasks);
+const data = new Data();
+data.setTasks(storage.loadData());
+
+window.addEventListener('beforeunload', () => {
+  storage.saveData(data.getTasks());
+});
```

Comment:
- [ ] By default, new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5)

![image](https://github.com/MoyasiGinko/ToDoList-Project/assets/75126481/2da39edb-2769-458c-bb7d-be2fb582478b)

Reply:
Hello @hiromataba , The new task added to the array already has the value completed: false by default. Please check again.